Files irrlicht-1.8.orig/source/Irrlicht/.CZipReader.cpp.swp and irrlicht-1.8/source/Irrlicht/.CZipReader.cpp.swp differ
diff -Nru -x '*~' irrlicht-1.8.orig/source/Irrlicht/CMakeLists.txt irrlicht-1.8/source/Irrlicht/CMakeLists.txt
--- irrlicht-1.8.orig/source/Irrlicht/CMakeLists.txt	1970-01-01 01:00:00.000000000 +0100
+++ irrlicht-1.8/source/Irrlicht/CMakeLists.txt	2013-04-12 01:01:43.269127300 +0200
@@ -0,0 +1,253 @@
+project(irrlicht)
+cmake_minimum_required(VERSION 2.8)
+
+set (IRRLICHT_SRCS
+    ${CMAKE_CURRENT_SOURCE_DIR}/C3DSMeshFileLoader.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CAnimatedMeshHalfLife.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CAnimatedMeshMD2.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CAnimatedMeshMD3.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CAnimatedMeshSceneNode.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CAttributes.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CB3DMeshFileLoader.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CBSPMeshFileLoader.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CBillboardSceneNode.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CBoneSceneNode.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CBurningShader_Raster_Reference.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CCSMLoader.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CCameraSceneNode.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CCgMaterialRenderer.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CColladaFileLoader.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CColladaMeshWriter.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CColorConverter.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CCubeSceneNode.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CD3D8Driver.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CD3D8NormalMapRenderer.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CD3D8ParallaxMapRenderer.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CD3D8ShaderMaterialRenderer.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CD3D8Texture.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CD3D9CgMaterialRenderer.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CD3D9Driver.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CD3D9HLSLMaterialRenderer.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CD3D9NormalMapRenderer.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CD3D9ParallaxMapRenderer.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CD3D9ShaderMaterialRenderer.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CD3D9Texture.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CDMFLoader.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CDefaultGUIElementFactory.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CDefaultSceneNodeAnimatorFactory.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CDefaultSceneNodeFactory.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CDepthBuffer.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CDummyTransformationSceneNode.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CEmptySceneNode.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CFPSCounter.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CFileList.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CFileSystem.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CGUIButton.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CGUICheckBox.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CGUIColorSelectDialog.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CGUIComboBox.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CGUIContextMenu.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CGUIEditBox.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CGUIEnvironment.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CGUIFileOpenDialog.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CGUIFont.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CGUIImage.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CGUIImageList.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CGUIInOutFader.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CGUIListBox.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CGUIMenu.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CGUIMeshViewer.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CGUIMessageBox.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CGUIModalScreen.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CGUIScrollBar.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CGUISkin.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CGUISpinBox.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CGUISpriteBank.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CGUIStaticText.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CGUITabControl.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CGUITable.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CGUIToolBar.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CGUITreeView.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CGUIWindow.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CGeometryCreator.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CImage.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CImageLoaderBMP.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CImageLoaderDDS.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CImageLoaderJPG.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CImageLoaderPCX.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CImageLoaderPNG.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CImageLoaderPPM.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CImageLoaderPSD.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CImageLoaderRGB.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CImageLoaderTGA.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CImageLoaderWAL.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CImageWriterBMP.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CImageWriterJPG.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CImageWriterPCX.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CImageWriterPNG.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CImageWriterPPM.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CImageWriterPSD.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CImageWriterTGA.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CIrrDeviceConsole.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CIrrDeviceFB.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CIrrDeviceLinux.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CIrrDeviceSDL.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CIrrDeviceStub.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CIrrDeviceWin32.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CIrrDeviceWinCE.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CIrrMeshFileLoader.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CIrrMeshWriter.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CLMTSMeshFileLoader.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CLWOMeshFileLoader.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CLightSceneNode.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CLimitReadFile.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CLogger.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CMD2MeshFileLoader.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CMD3MeshFileLoader.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CMS3DMeshFileLoader.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CMY3DMeshFileLoader.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CMemoryFile.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CMeshCache.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CMeshManipulator.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CMeshSceneNode.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CMetaTriangleSelector.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CMountPointReader.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CNPKReader.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CNullDriver.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/COBJMeshFileLoader.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/COBJMeshWriter.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/COCTLoader.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/COSOperator.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/COctreeSceneNode.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/COctreeTriangleSelector.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/COgreMeshFileLoader.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/COpenGLCgMaterialRenderer.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/COpenGLDriver.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/COpenGLExtensionHandler.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/COpenGLNormalMapRenderer.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/COpenGLParallaxMapRenderer.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/COpenGLSLMaterialRenderer.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/COpenGLShaderMaterialRenderer.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/COpenGLTexture.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CPLYMeshFileLoader.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CPLYMeshWriter.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CPakReader.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CParticleAnimatedMeshSceneNodeEmitter.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CParticleAttractionAffector.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CParticleBoxEmitter.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CParticleCylinderEmitter.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CParticleFadeOutAffector.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CParticleGravityAffector.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CParticleMeshEmitter.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CParticlePointEmitter.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CParticleRingEmitter.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CParticleRotationAffector.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CParticleScaleAffector.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CParticleSphereEmitter.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CParticleSystemSceneNode.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CQ3LevelMesh.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CQuake3ShaderSceneNode.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CReadFile.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CSMFMeshFileLoader.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CSTLMeshFileLoader.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CSTLMeshWriter.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CSceneCollisionManager.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CSceneLoaderIrr.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CSceneManager.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CSceneNodeAnimatorCameraFPS.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CSceneNodeAnimatorCameraMaya.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CSceneNodeAnimatorCollisionResponse.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CSceneNodeAnimatorDelete.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CSceneNodeAnimatorFlyCircle.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CSceneNodeAnimatorFlyStraight.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CSceneNodeAnimatorFollowSpline.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CSceneNodeAnimatorRotation.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CSceneNodeAnimatorTexture.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CShadowVolumeSceneNode.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CSkinnedMesh.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CSkyBoxSceneNode.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CSkyDomeSceneNode.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CSoftwareDriver.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CSoftwareDriver2.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CSoftwareTexture.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CSoftwareTexture2.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CSphereSceneNode.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CTRFlat.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CTRFlatWire.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CTRGouraud.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CTRGouraud2.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CTRGouraudAlpha2.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CTRGouraudAlphaNoZ2.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CTRGouraudWire.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CTRNormalMap.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CTRStencilShadow.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CTRTextureBlend.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CTRTextureDetailMap2.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CTRTextureFlat.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CTRTextureFlatWire.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CTRTextureGouraud.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CTRTextureGouraud2.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CTRTextureGouraudAdd.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CTRTextureGouraudAdd2.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CTRTextureGouraudAddNoZ2.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CTRTextureGouraudAlpha.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CTRTextureGouraudAlphaNoZ.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CTRTextureGouraudNoZ.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CTRTextureGouraudNoZ2.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CTRTextureGouraudVertexAlpha2.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CTRTextureGouraudWire.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CTRTextureLightMap2_Add.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CTRTextureLightMap2_M1.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CTRTextureLightMap2_M2.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CTRTextureLightMap2_M4.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CTRTextureLightMapGouraud2_M4.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CTRTextureWire2.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CTarReader.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CTerrainSceneNode.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CTerrainTriangleSelector.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CTextSceneNode.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CTriangleBBSelector.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CTriangleSelector.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CVideoModeList.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CVolumeLightSceneNode.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CWADReader.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CWaterSurfaceSceneNode.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CWriteFile.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CXMLReader.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CXMLWriter.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CXMeshFileLoader.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CZBuffer.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/CZipReader.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/IBurningShader.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/Irrlicht.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/irrXML.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/os.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/lzma/LzmaDec.c
+    ${CMAKE_CURRENT_SOURCE_DIR}/aesGladman/aescrypt.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/aesGladman/aeskey.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/aesGladman/aestab.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/aesGladman/fileenc.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/aesGladman/hmac.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/aesGladman/prng.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/aesGladman/pwd2key.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/aesGladman/sha1.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/aesGladman/sha2.cpp
+)
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}/../../include)
+include_directories(${CMAKE_INCLUDE_PATH})
+find_package(zlib REQUIRED)
+find_package(jpeg REQUIRED)
+find_package(png REQUIRED)
+find_package(bzip2 REQUIRED)
+
+
+add_definitions(-DIRRLICHT_EXPORTS=1)
+add_library(irrlicht SHARED ${IRRLICHT_SRCS})
+install(TARGETS irrlicht RUNTIME DESTINATION bin
+                         LIBRARY DESTINATION lib
+                         ARCHIVE DESTINATION lib)
+                      target_link_libraries(irrlicht "-lgdi32 -lopengl32 -ld3dx9d -lwinmm -Wl,--add-stdcall-alias")
+                      target_link_libraries(irrlicht ${ZLIB_LIBRARIES})
+                      target_link_libraries(irrlicht ${PNG_LIBRARIES})
+                      target_link_libraries(irrlicht ${JPEG_LIBRARIES})
+                      target_link_libraries(irrlicht ${BZIP2_LIBRARIES})
diff -Nru -x '*~' irrlicht-1.8.orig/source/Irrlicht/CZipReader.cpp irrlicht-1.8/source/Irrlicht/CZipReader.cpp
--- irrlicht-1.8.orig/source/Irrlicht/CZipReader.cpp	2013-04-12 01:06:54.462926500 +0200
+++ irrlicht-1.8/source/Irrlicht/CZipReader.cpp	2013-04-12 01:03:28.432142300 +0200
@@ -29,13 +29,7 @@
 	#ifdef _IRR_COMPILE_WITH_ZIP_ENCRYPTION_
 	#include "aesGladman/fileenc.h"
 	#endif
-	#ifdef _IRR_COMPILE_WITH_BZIP2_
-	#ifndef _IRR_USE_NON_SYSTEM_BZLIB_
 	#include <bzlib.h>
-	#else
-	#include "bzip2/bzlib.h"
-	#endif
-	#endif
 	#ifdef _IRR_COMPILE_WITH_LZMA_
 	#include "lzma/LzmaDec.h"
 	#endif
